src/Lachain.Core/RPC/HTTP/Web3/BlockchainServiceWeb3.cs

```diff
@@ -668,6 +672,32 @@ private JArray GetCurrentValidators()
             }
             return validators;
         }
+        
+        [JsonRpcMethod("la_consensusState")]
+        private JArray GetConsensusState(long era)
```

Comment:
This call should return all currently running protocol instances,  current implementation returns only protocols for given era and user hasn't any calls to understand what era he needs,